keyboards/enviousdesign/65m/info.json

```
@@ -0,0 +1,103 @@
+{
+    "manufacturer": "Envious.Design",
+    "keyboard_name": "EnvKB 65M",
+    "maintainer": "Envious.Design",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["GP0", "GP1", "GP2", "GP3", "GP4", "GP5", "GP6", "GP7", "GP8", "GP9", "GP10", "GP11", "GP12", "GP13", "GP14", "GP15", "GP16"],
+        "rows": ["GP21", "GP20", "GP19", "GP18", "GP17"]
+    },
+    "processor": "RP2040",
+    "url": "http://envious.design",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0xA279",
+        "vid": "0xE739"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Looks like multiple layouts are supported; this should be `LAYOUT_all`.

```suggestion
        "LAYOUT_all": {
```